azurelinuxagent/ga/exthandlers.py

```diff
@@ -674,7 +674,7 @@ def launch_command(self, cmd, timeout=300):
             raise ExtensionError("Failed to launch: {0}, {1}".format(cmd, e))
 
         retry = timeout / 5
-        while retry > 0 and child.poll == None:
+        while retry > 0 and child.poll is None:
```

Comment:
@hglkrijger I still don't believe this code is correct. `child.poll` is a method. It returns immediately after setting `child.returncode`. If `child.returncode is None` is true, then the child is still active. Otherwise it is terminated.